x-pack/plugins/lists/public/exceptions/hooks/use_exception_list.tsx

```diff
@@ -4,12 +4,12 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { useEffect, useState } from 'react';
+import { useCallback, useEffect, useState } from 'react';
 
 import { fetchExceptionListById, fetchExceptionListItemsByListId } from '../api';
 import { ExceptionListAndItems, UseExceptionListProps } from '../types';
 
-export type ReturnExceptionListAndItems = [boolean, ExceptionListAndItems | null];
+export type ReturnExceptionListAndItems = [boolean, ExceptionListAndItems | null, () => void];
 
 /**
  * Hook for using to get an ExceptionList and it's ExceptionListItems
```

Comment:
JSDoc update here with the new params:

```suggestion
/**
 * Hook for using to get an ExceptionList and it's ExceptionListItems
 *
 * @param http
 * @param id desired ExceptionList ID (not list_id)
 *
 * @param namespaceType
 * @param pagination
 * @param filterOptions
 * @param onError
 */
```

Reply:
Thanks for pointing that out! Updated the docs for all these.